tests/unit-tests/test-class-utils.php

```diff
@@ -519,4 +519,77 @@ public function testGetTargetResumeId_WhenCalled_ReturnsCourseIdIfItHasNoLessons
 		/* Assert */
 		$this->assertEquals( $course_id, $result );
 	}
+
+	public function testIsFseThemeSupported_WhenCalled_WorksIfBlockTemplatesIndexAvailable() {
+		/* Arrange */
+		$theme_directory = get_template_directory() . '/block-templates';
+		$index_file      = $theme_directory . '/index.html';
+
+		// Remove the 'block-templates/index.html' file if it exists.
+		if ( file_exists( $index_file ) ) {
+			unlink( $index_file );
+		}
+
+		// Create the 'block-templates' directory if it doesn't exist.
+		if ( ! is_dir( $theme_directory ) ) {
+			mkdir( $theme_directory );
+		}
+
+		$this->create_file( $index_file );
+
+		/* Assert */
+		// Call the function and assert the result.
+		$this->assertTrue( Sensei_Utils::is_fse_theme() );
+
+		unlink( $index_file );
+		rmdir( $theme_directory );
+	}
+
+	public function testIsFseThemeSupported_WhenCalled_WorksIfIndexHtmlAvailable() {
```

Comment:
I think we should improve this name. The condition must be in the middle. And the result ("Works" in this case) is too vague.

```suggestion
	public function testIsFseTheme_WhenIndexHtmlAvailable_ReturnsTrue() {
```

Just want to remind the general schema for a test name:
`test[MethodUnderTest]_[Conditions]_[Result]`